gh-actions/commit-size/action.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'Commit Size'
+description: 'Make sure no commit in the pull request goes over a certain size'
+inputs:
+  size:
+    description: 'Total maximum size of commit (lines added+removed).'
+    required: false
+    default: '500'
+runs:
+  using: "composite"
+  steps:
+    - name: Make sure the size of each commit doesn't pass the desired size
+      shell: bash
+      run: |
+        set -e
+        git fetch origin refs/pull/${{ github.event.pull_request.number}}/head
+        commits_range=${{ github.event.pull_request.base.sha }}..${{ github.event.pull_request.head.sha }}
+        sum_commit() { git diff --numstat ${1} ${1}^ | cut -f1,2 -d$'\t' | tr $'\t' '+' | paste -sd+ - | bc; }
```

Comment:
```suggestion
        sum_commit() { git diff --numstat ${1} ${1}^ | awk '{ sum += $1 + $2 } END { print sum }; }
```

Reply:
Thanks, you were missing a closing quote so I applied manually and force-pushed an updated version.

Reply:
Ah yes, apologies!